File: apps/android/app/src/main/java/com/mecatrack/mobile/domain/WorkOrderStatusLabel.kt
package com.mecatrack.mobile.domain

fun workOrderStatusLabel(status: String): String =
    when (status) {
        "EN_PROCESO" -> "En proceso"
        "LISTA_PARA_ENTREGA" -> "Lista para entrega"
        "OWNER_CONTACTED" -> "Propietario contactado"
        else -> status
    }

File: apps/api/src/modules/history/mappers/work-order-status-label.ts
import { WorkOrderStatus } from '@prisma/client';

const STATUS_LABELS: Record<WorkOrderStatus, string> = {
  EN_PROCESO: 'En proceso',
  LISTA_PARA_ENTREGA: 'Lista para entrega',
  OWNER_CONTACTED: 'Propietario contactado',
  ENTREGADA: 'Entregada',
};

export function toStatusLabel(status: WorkOrderStatus): string {
  return STATUS_LABELS[status];
}
